static/performance/syscall.csv

```diff
@@ -0,0 +1,4 @@
+runtime,metric,result
+runc,syscall_time_ns,1929.0

```

Comment:
This seems way too high. This is on broadwell? It should be ~300ns.

Reply:
@prattmic If you think changes are required could you request changes as part of your review?

Reply:
I don't think it necessarily needs changes. Just something to look into.

Reply:
We're using getpid as the call. I believe that there's actually a material hit from namespaces.

Reply:
I thought we switched to gettimeofday?

Reply:
As discussed offline, this is likely overhead from the BPF filters for the default security profile. This since is the standard for docker out of the box, we will leave it for now.